CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Unreleased
 
+- Drop runtime error when a mutable `TypedDict` key overrides a read-only
```

Comment:
This comment is back to front I think @JelleZijlstra ? May be confusing

Reply:
You're right, thanks